java/lang/Float/Float.hpp

```diff
@@ -403,39 +383,39 @@ namespace Java {
 			string toString() const;
 
 			/**
-             * Returns a string representation of the {double}
+             * Returns a string representation of the float
              * argument. All characters mentioned below are ASCII characters.
              *
-             * @param   d   the {double} to be converted.
+             * @param   d   the float to be converted.
              * @return a string representation of the argument.
              */
 			static String toString(float d);

```

Comment:
Please rename param name and check for others

Reply:
Thanks. Fixed